### DISCUSSION TITLE
Label préparation graphique

### DISCUSSION THREAD

**Discussion by Synfactory** [witsa/synapps#734] [Q&A] [answered]
Bonjour,
vous sauriez où on peut trouver les labels des préparation graphique dans le REDY?
![1](https://github.com/witsa/synapps/assets/165296029/43dc6c07-cc4d-4e83-9755-9427fe7828b9)

Merci

**Comment by manufitoussiwit** [comment, marked as answer]
Bonjour,

Elles sont ici dans le REDY : 

![image](https://github.com/witsa/synapps/assets/35595723/d4c52f51-2be6-49e9-8945-0da2d6f8308a)

**Reply by Synfactory** [reply]
Merci, j'ai cherché à tort dans WEB/RIA...
